src/App.css

```diff
@@ -169,4 +173,12 @@ span {
 
 .fn-wg-600{
 	font-weight: 600;
+}
+
+.handle-img-np{
+    height: 2.5em;
+    width: 2.5em;
+    border-radius: 50%;
+    background-color: var(--primary-color);
+    color: white;
```

Comment:
use color variables

Reply:
i guess this are already variable provide by CSS i guess creating variable for default color is redoing stuff